services/rook-ceph-cluster/1.10.1/defaults/cm.yaml

```
@@ -30,7 +30,7 @@ data:
         volumeClaimTemplate:
           spec:
             # Use the default storage class configured in cluster.
-            storageClassName: ""
+            # not setting storageClass to let it fall to environment default
```

```suggestion
            # not setting storageClass to let it fall to environment default
            # storageClassName: ""
```
Might still be helpful to include the field just commented out